README.rst

```diff
@@ -82,6 +126,18 @@ otherwise noted.
 
 Please see `LICENSE.txt <LICENSE.txt>`_ for details.
 
+Areas of concern/refinement
+*******
+
+This is very preliminary work proving out our ability to confirm and control Django 
+signal / receiver mappings using the settings file. It's not a fully robust implementation 
+and is just a guidepost showing that our intended implementation can work.
+
+We should consider whether we really want to allow signals to be sent in the Celery or 
+management command environments; it may be easier to reason about if signals are only 
+confined to the primary IDA and other environments could call API endpoints to trigger 
+workflows.
+
```

Comment:
The [original PR](https://github.com/edx/commerce-coordinator/pull/7) had two suggestions, but I left them out since they're currently in progress ([REV-2483](https://openedx.atlassian.net/browse/REV-2483) and [REV-2484](https://openedx.atlassian.net/browse/REV-2484))